src/main/java/org/kohsuke/github/GHCommit.java

```diff
@@ -38,12 +34,43 @@
 
         private int comment_count;
 
+        private String id;
+        private String timestamp;
+        private String treeId;
+
         static class Tree {
             String sha;
         }
 
         private Tree tree;
 
+        /**
+         * Gets id of the commit, used by {@link GHCheckSuite} when a {@link GHEvent#CHECK_SUITE} comes
+         *
+         * @return id of the commit
+         */
+        public String getId() {
+            return id;
+        }
+
+        /**
+         * Gets timestamp of the commit, used by {@link GHCheckSuite} when a {@link GHEvent#CHECK_SUITE} comes
+         *
+         * @return timestamp of the commit
+         */
+        public Date getTimestamp() {
```

Comment:
I don't see it on this page: https://developer.github.com/v3/repos/commits/#get-a-single-commit 

I see this:
https://developer.github.com/v3/activity/events/types/#webhook-payload-example-1

It possible the "issue" in `GHCheckSuite` is not a `GHIssue`?  But instead something else?

Reply:
sry, didn't notice that [commit page](https://developer.github.com/v3/repos/commits/#get-a-single-commit) and pay much attention to the structure. I'll add it inside `GHCheckSuite`.